docs/app-publishing.md

```diff
@@ -0,0 +1,33 @@
+2. Sign into the Partner Center. You should now be able to reserve a name for your app ([Create your app by reserving a name - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/create-your-app-by-reserving-a-name)).
+3. Open your app's solution in Visual Studio.
+4. If you haven't already, generate visual assets for your app. You can generate visual assets from a base image in Visual Studio by opening your app's `Package.appmanifest` and navigating to the 'Visual Assets' section. Here you can supply your base image, and Visual Studio will generate the needed collection of assets.
+5. Right-click on your app project and select `Publish > Associate App with the Store…`. After you’ve signed-in (make sure you sign-in with the same account as the one used for the Partner Center) and selected your app, Visual Studio will update your `Package.appxmanifest` file with the new Publisher (a GUID) and the app name. You’ll also get a new `Package.StoreAssociation.xml` file in your project that should not be checked in ([Packaging MSIX apps - MSIX | Microsoft Docs](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#configure-your-project)).
+6. Set your solution configuration to 'Release'. Right-click on your app project and choose  `Publish > Create App Packages`. If you have already associated your project with an app in the Microsoft Store, you should have an option to 'Create Packages for the Associated Store App'. Fill out the form according to your app's information. This process should create an `.appxupload`, which you will upload to Partner Center later ([Packaging MSIX apps - MSIX | Microsoft Docs](https://docs.microsoft.com/en-us/windows/msix/package/packaging-uwp-apps#generate-an-app-package-upload-file-for-store-submission)).
+7. Run your app through the latest version of the Windows App Certification Kit, and verify that it passes all tests ([Windows App Certification Kit - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/windows-app-certification-kit)).
+8. Select your newly reserved app name from the 'Overview' section in Partner Center to begin adding information to your app.
+9. Locate the 'Start a Submission' button on the current webpage. Hitting this button will open up the form for your first submission to the app store. 
+10. The app submission form will contain many questions about your app's age ratings, properties, pricing, etc. Click on each of the sections to fill in the information for your app ([App submissions - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/app-submissions)):
+  
+  - Pricing and Availability ([Set app pricing and availability - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/set-app-pricing-and-availability))
```

Comment:
Do we really need to replicates the break down of each section here?

Reply:
Could you clarify your question for me? To clarify the links on my end, the links here take the user to docs on how to fill out each of those sections not take the user to that section of the submission form.